src/content/docs/apis/nerdgraph/examples/nerdgraph-slm.mdx

```diff
@@ -85,6 +85,57 @@ It contains these fields:
   * `timeWindow.rolling.count`: The length of the period taken into consideration to calculate the SLO. The supported values are `1`, `7`, `14`, `28`, and `30`.
   * `timeWindow.rolling.unit`: `DAY`is the supported value.
 
+### Using `SELECT` functions and wildcard
+
+The default select operation is a `COUNT`, but we also support `SUM` even for several metrics using a wildcard:
+
+```graphql
+mutation {
+  serviceLevelCreate(
+    entityGuid: "entityGuid"
+    indicator: {
+      name: "Success reqeust"
+      description: "The proportion of success requests count is 99% that the total count"
+      events: {
+        validEvents: { 
+          select: {
+            function: SUM
+            attribute: "http.request.status.%.count"
+          },
+          from: "Metric"
+        }
+        goodEvents: {
+          select: {
+            function: SUM
+            attribute: "http.request.status.2%.count"
+          },
+          from: "Metric"
+        }
+        accountId: accountId
+      }
+      objectives: {
+        target: 99.5
+        timeWindow: { 
+          rolling: { 
+            count: 7, 
+            unit: DAY 
+          } 
+        }
+      }
+    }
+  ) {
+    id
+    description
+  }
+}
+```
+
+Notice the properties `validEvents` and `goodEvents` of `events` now includes a `select`. In the select you can configure the function:
+- `COUNT`: default function, will count number of results;
+- `SUM`: sums all values for the selected events/metric.
+
+Another important differente in this example is the usage of wildcard (`%`) to query values of all metrics with the same prefix. Imagine we are reporting request count by status (e.g. `http.request.status.200.count`, `http.request.status.201.count`, `http.request.status.400.count`, ect...), the query will sum all matching metric name using the wildcard.
```

Comment:
Instead of `Imagine we are reporting request` we could say `Imagine your application is reporting ...`, wdyt?